### DISCUSSION TITLE
can i draw imgui menu by module?

### DISCUSSION THREAD

**Discussion by cry-1337** [SmartlyDressedGames/Unturned-3.x-Community#4291] [Q&A] [answered]
how can i do this if its possible

**Comment by DanielWillett** [comment, marked as answer]
You can by attaching a SleekFullscreenBox (or any element really) to one of the base windows like PlayerUI.window, EditorUI.window, etc.

(shameless plug) I suggest my UI extension library for modules that lets you attach UI classes to any existing UI types in the base game (including sleek wrappers) https://github.com/DanielWillett/UnturnedUITools.

![68747470733a2f2f692e696d6775722e636f6d2f4a68593948424a2e706e67](https://github.com/SmartlyDressedGames/Unturned-3.x-Community/assets/12886600/2b696930-8e57-47ef-b7d2-29344745e16f)

**Reply by Jdance-Media** [reply]
I use the UIExtensions library as well. Just be aware that the Sleek UI stuff only works currently on clientside modules. 

Also like to add that we are on uGUI and UIToolkit now, IMGUI is obsolete.
